src/SprykerEco/Yves/Payone/PayoneFactory.php

```diff
@@ -55,23 +64,23 @@ class PayoneFactory extends AbstractFactory
     /**
      * @return \SprykerEco\Yves\Payone\Form\PrePaymentForm
      */
-    public function createPrePaymentForm()
+    public function createPrePaymentForm(): PrePaymentForm
```

Comment:
* The return type should be an interface or an abstarct class.

Reply:
++